.github/workflows/integration.yml

```diff
@@ -66,3 +62,10 @@ jobs:
         run: |
           berty-integration \
             -integration.testbot=`cat ./config/config.gen.json | jq -r '.berty.contacts["testbot"].link'`
+      - name: Discord notification
+        if: github.event_name == 'schedule' && failure()
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_DEV_WEBHOOK }}
+        uses: Ilshidur/action-discord@0.3.0
+        with:
+          args: 'Integration failed, see https://github.com/berty/berty/actions?query=workflow%3AIntegration.'
```

Comment:
```suggestion
      - name: Run integration tests with production config
        id: run
        run: |
          berty-integration \
            -integration.testbot=`cat ./config/config.gen.json | jq -r '.berty.contacts["testbot"].link'`
          echo "::set-output name=runUrl::https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
      - name: Discord notification
        if: github.event_name == 'schedule' && failure()
        env:
          DISCORD_WEBHOOK: ${{ secrets.DISCORD_DEV_WEBHOOK }}
        uses: Ilshidur/action-discord@0.3.0
        with:
          args: 'Integration failed, see '${{ steps.run.outputs.runUrl }}
```

Reply:
good idea, you can open a new PR :+1:

Reply:
Done https://github.com/berty/berty/pull/2670/